lib/tdf3/src/client/index.ts

```diff
@@ -261,6 +261,8 @@ export class Client {
     payloadKey,
   }: EncryptParams): Promise<AnyTdfStream | null> {
     if (rcaSource && asHtml) throw new Error('rca links should be used only with zip format');
+    if (rcaSource && !this.kasEndpoint)
```

Comment:
Is this a valid assumption? Since RCA links will need a policy uuid ?

Reply:
It's fine to have this error here, if only as a precaution